pkg/api/annotations/annotations.go

```diff
@@ -16,8 +16,35 @@ limitations under the License.
 
 package annotations
 
-const kubectlPrefix = "kubectl.kubernetes.io/"
+const (
```

Comment:
I think it would be nice to have a unified place to put all API related annotation constants that everyone could use. Put it in a package makes that easier.

The weird thing for annotation_key_constants.go is that we have created two of them. Is it really necessary to distinguish them for v1 and internal?